src/System.Runtime.TimeZoneData/Versions.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <TimeZoneDataVersion>2021a</TimeZoneDataVersion>
```

Comment:
Why encapsulate this into a separate file?

Reply:
Ah just saw the xml update. I would just keep that property in the project file and do a regex replace of `<TimeZoneDataVersion>.*</TimeZoneDataVersion>`. An alternative would be to move that property into eng\Versions.props

Reply:
because I'm editing it with xmlpoke and it destroyed the formatting of the csproj if it was selfmodifying

Reply:
I moved the version back in to the csproj and wrote some terrible replacement code that should at least do a functional in place update.